manifests/backup/mysqlbackup.pp

```diff
@@ -5,7 +5,7 @@
   $backupdir,
   $backupdirmode = '0700',
   $backupdirowner = 'root',
-  $backupdirgroup = 'root',
+  $backupdirgroup = $mysql::params::root_group,
```

Comment:
Well, we *might* use `$mysql::root:group` here, but this module's style is to use `$mysql::params::*` elsewhere.